### ISSUE TITLE
Metrics exporter throws warnings

### ISSUE THREAD

**Issue by atoulme** [open-telemetry/opentelemetry-java-instrumentation#1381]
I am confused by the following warning that appears in my logs:
```[opentelemetry.auto.trace 2020-10-13 17:26:25:919 +0000] [grpc-default-executor-1] WARN io.opentelemetry.exporters.otlp.OtlpGrpcMetricExporter - Failed to export metrics
io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: deadline exceeded after -7532692200ns. [buffered_nanos=2235161800, waiting_for_connection]
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:515)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

Is this library exporting metrics?
Which metrics?
Are those internal metrics to the agent or is it catching on metrics exposed over JMX or other mechanisms?
What is the expectation of the metric exporter, which target does it run with by default?
My traces make it over to the collector. Do I need to set up an oltp metric receiver as well, always? Can I disable the metric exporter?

**Comment by iNikem** [comment]
@atoulme  What version of instrumentation javaagent are you using?

**Comment by atoulme** [comment]
I use version 0.8.0.

I most likely did not set up the collector to collect metrics. But I'm not sure what metrics are being exported or why this tracing instrumentation exports metrics. Any doc?

**Comment by trask** [comment]
hey @atoulme! there are a few [self-diagnostic metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/error-handling.md#self-diagnostics) emitted currently

we are planning to capture other metrics, e.g. #1309, and tracing-based metrics in the future

if you don't want metrics exported, you can specify `OTEL_EXPORTER=otlp_span` starting in the just released `0.9.0`

**Comment by atoulme** [comment]
Thanks @trask that's helpful. Ideally I guess I'd like those metrics too. Does this mean I need to consider using the same pipeline for both metrics and traces?

**Comment by RashmiRam** [comment]
> hey @atoulme! there are a few [self-diagnostic metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/error-handling.md#self-diagnostics) emitted currently
> 
> we are planning to capture other metrics, e.g. #1309, and tracing-based metrics in the future
> 
> if you don't want metrics exported, you can specify `OTEL_EXPORTER=otlp_span` starting in the just released `0.9.0`

@trask This doesn't seem to work for us. It showed no such exporter found exception. We are using 0.9.0. We have done this https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1111#issuecomment-715389658 as stop gap to avoid too much logs.

**Comment by trask** [comment]
hey @RashmiRam! sorry, we didn't quite get the setting right in 0.9.0, this is fixed in `master` and will be in 0.10.0 late this week or next week.

**Comment by iNikem** [comment]
Will be closed by #1393
